src/cast_core/src/CollectionScanner.cpp

```diff
@@ -107,7 +107,7 @@ struct CollectionScanner::PhaseConfig {
         } else if (scanTypeString == "snapshot") {
             transactionOptions = mongocxx::options::transaction{};
             auto readConcern = mongocxx::read_concern{};
-            readConcern.acknowledge_level(mongocxx::read_concern::level::k_majority);
+            readConcern.acknowledge_level(mongocxx::read_concern::level::k_snapshot);
```

Comment:
Necessary to prevent yielding on the `getMore`.

Reply:
Consider adding this comment to the source for the next maintainer 😄